docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,211 @@
+---
+id: getting-started
+title: Getting Started on Linux
+layout: docs
+category: Quick Start
+permalink: docs/getting-started-on-linux.html
+next: android-setup
+---
+
+This guide is essentially a beginner-friendly version of the [Getting Started](/react-native/docs/getting-started.html) page for React Native on Linux.
+
+### Platform
+
+For the purposes of this guide, we assume that you're working on Ubuntu Linux 14.04 LTS.
+
+### Prerequisites
+
+Before following this guide, you should have installed the Android SDK and run a successful Java-based "Hello World" app for Android.
+
+See [Android Setup](/react-native/docs/android-setup.html) for details.
+
+#### Installing NodeJS
+
+The first thing you need to do is to install NodeJS, a popular Javascript implementation.
+
+Fire up a terminal and paste these commands in to install NodeJS from APT:
```

Comment:
```
Fire up the Terminal and paste the following commands to install NodeJS from the [NodeSource](https://nodesource.com/) repository:
```

Also it would be great if we could include instructions for other popular distros, like Fedora and Arch. Or maybe just link to the instructions page for other distros.

Prolly also include this command `ln -s /usr/bin/nodejs /usr/bin/node` to symlink `node` to `nodejs`.